config/environments/development.rb

```diff
@@ -22,6 +25,9 @@
     memcached_servers = YAML.load_file(Rails.root.join("config/local.yml")).fetch("MEMCACHED_SERVERS", memcached_servers) if File.file?(Rails.root.join("config/local.yml"))
     config.cache_store = :mem_cache_store, memcached_servers,
                          { namespace: "ao3-v2-dev", compress: true, pool_size: 10 }
+    config.public_file_server.headers = {
```

Comment:
Google isn't being my friend on this -- is this a default?

Reply:
It is: https://github.com/railsdiff/rails-new-output/blob/cb6a93be67c30040485603cf5b51c7dae4144251/config/environments/development.rb#L27. (That repository powers https://railsdiff.org/ and both are very helpful for comparing config changes between different Rails versions.)